x-pack/platform/plugins/shared/onechat/public/application/components/conversations/conversation_input/conversation_input_form.tsx

```diff
@@ -50,9 +59,13 @@ export const ConversationInputForm: React.FC<ConversationInputFormProps> = ({ on
     }
   `;
 
+  const formContainerDisabledStyles = css`
+    background-color: ${euiTheme.colors.backgroundBaseDisabled};
+  `;
```

Comment:
Not a big fan of this approach, this is a workaround due to the structure of the components within this container. Ideally, we should just disable the text area but due to the styling of other components around it, it wasn't straight forward. I decided not to refactor them in this PR as I know we're planning on moving to a more rich text editor soon? Lmk your thoughts @zacharyparikh